config/dynamodb.js

```diff
@@ -29,6 +29,48 @@ const tableProps = {
             },
         ],
     },
+    'audios': {
+        AttributeDefinitions: [
+            {
+                AttributeName: 'url',
+                AttributeType: 'S',
+            },
+            {
+                AttributeName: 'time',
+                AttributeType: 'N',
+            },
+            {
+                AttributeName: 'date',
+                AttributeType: 'S',
+            },
+        ],
```

Comment:
DynamoDB supports a field type `Date`. Rather use that than different fields.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html

Reply:
If the range query works better with numbers, consider using unix epoch integers.